test/relayer_chain_test.go

```diff
@@ -115,7 +115,11 @@ func chainTest(t *testing.T, tcs []testChain) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	_ = relayer.StartRelayer(ctx, src, dst, 2*cmd.MB, 5)
+	filter := &relayer.ChannelFilter{
+		Rule:        "",
+		ChannelList: nil,
+	}
+	_ = relayer.StartRelayer(ctx, src, dst, filter, 2*cmd.MB, 5)
 	require.NoError(t, err)
```

Comment:
I'm reviewing from my phone, and I can't see what error is being checked here. The check should move closer to the error's assignment.